test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+${PHANTOM_JS}/bin/phantomjs --webdriver=4444&
```

Comment:
I have another idea for this that I'd like to add to this PR this is starting to feel messy that we add phantom like this